helm/templates/gateway/gateway-configmap.yaml

```diff
@@ -355,7 +355,13 @@ data:
     reporters:
       # Elasticsearch reporter
       elasticsearch:
+        {{- if hasKey .Values.gateway "reporters" }}
+        {{- if hasKey .Values.gateway.reporters "elasticsearch" }}
+        {{- if hasKey .Values.gateway.reporters.elasticsearch "enabled" }}
         enabled: {{ .Values.gateway.reporters.elasticsearch.enabled }}
+        {{- end }}
+        {{- end }}
+        {{- end }}
```

Comment:
@sebdevaux I'd like your advice on this one (+ the changes on the values.yaml)

Reply:
I have FP this to simplify:

```
        {{- if (hasKey ((.Values.gateway).reporters).elasticsearch "enabled") }}
        enabled: {{ .Values.gateway.reporters.elasticsearch.enabled }}
        {{- end }}

```